.env.example

```diff
@@ -24,3 +24,5 @@ GITLAB_ACCOUNT_TOKEN=gitlab-account-token
 DRY_RUN=true
 # Boolean to force all repos to re-sync, regardless of whether they are already fully mirrored. Can be used if e.g. we wish to add additional tags during the mirror process. Should generally be set to false, or removed altogether
 FORCE_UPDATE=false
+# Boolean to indicate dry run mode for major version tag script
+TAGGER_DRY_RUN=true
```

Comment:
I'm not sure this is needed, since in the GitHub workflow YAML we can have `DRY_RUN` set for this script and not for the mirror.

Reply:
I'm not sure I follow. Do you mean we could hardcode the DRY_RUN value into the GitHub workflow YAML? My intention was that we should be able to switch either the tagging or the mirroring into dry run mode just by editing the appropriate secret, and without any code changes.

Reply:
Oh, OK. I was thinking this was just a temporary thing, but if you think we might need to switch the scripts on/off in the distant future then sure, but in that case we should make the naming more consistent:

```shell
MIRROR_DRY_RUN=...
TAGGER_DRY_RUN=...
```

or `DRY_RUN_<script>` etc.

Reply:
I've renamed them all now, so this should be ready for another look.